ajax/libs/imgareaselect/package.json

```diff
@@ -26,5 +26,13 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/odyniec/imgareaselect"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/odyniec/imgareaselect.git",
+    "basePath": "distfiles",
+    "files": [
+      "**!(*.txt)/*"
```

Comment:
`jquery.min.js` should also been excluded, it's such a well known library been hosted everywhere, and also on cdnjs. We don't need to host it here.